src/api/fareContracts.ts

```diff
@@ -21,7 +21,7 @@ export async function search(
   params: OfferSearchParams,
   opts?: AxiosRequestConfig,
 ): Promise<Offer[]> {
-  const url = 'ticket/v1/search';
+  const url = 'ticket/v1/search/zones';
```

Comment:
I would have preferred:
```suggestion
  const url = 'ticket/v1/search/' + 'z' + 'o' + 'n' + 'e' + 's';
```

Reply:
That is what I call _adding value_!